official-allow_functionality.json

```diff
@@ -229,8 +229,43 @@
 		{"o":{"h":"*.codepen.io"},"d":{"h":"*.s3-us-west-2.amazonaws.com"}},
 		{"o":{"h":"*.codepen.io"},"d":{"h":"*.cloudflare.com"}},
 		{"o":{"h":"*.hansje.net"},"d":{"h":"*.cargocollective.com"}},
-		{"o":{"h":"*.indiegogo.com"},"d":{"h":"*.iggcdn.com"}}
+		{"o":{"h":"*.indiegogo.com"},"d":{"h":"*.iggcdn.com"}},
+		{"o":{"h":"*.eurobuch.com"},"d":{"h":"*.lehmanns.de"}},
+		{"o":{"h":"*.emp.de"},"d":{"h":"*.acfrg.com"}},
+		{"o":{"h":"*.microsoft.com"},"d":{"h":"*.s-msft.com"}},
+		{"o":{"h":"*.eff.org"},"d":{"h":"*.trackersimulator.org"}},
+		{"o":{"h":"*.eff.org"},"d":{"h":"*.eviltracker.net"}},
+		{"o":{"h":"*.eff.org"},"d":{"h":"*.do-not-tracker.org"}},
+		{"o":{"h":"*.shortfil.ms"},"d":{"h":"*.eay.cc"}},
+		{"o":{"h":"*.virustotal.com"},"d":{"h":"*.virustotalcloud.appspot.com"}},
+		{"o":{"h":"*.upb.de"},"d":{"h":"*.pingo.cc"}},
+		{"o":{"h":"*.az-online.de"},"d":{"h":"*.idcdn.de"}},
+		{"o":{"h":"*.emp.de"},"d":{"h":"*.cachefly.net"}},
+		{"o":{"h":"*.norwegianrain.com"},"d":{"h":"*.shopify.com"}},
+		{"o":{"h":"*.sinn.de"},"d":{"h":"*.bildsequenz.de"}},
+		{"o":{"h":"*.expedia.de"},"d":{"h":"*.travel-assets.com"}},
+		{"o":{"h":"*.expedia.de"},"d":{"h":"*.trvl-media.com"}},
+		{"o":{"h":"*.lonelyplanet.de"},"d":{"h":"*.123poi.com"}},
+		{"o":{"h":"*.codehunt.com"},"d":{"h":"*.msecnd.net"}},
+		{"o":{"h":"*.dreamtheater.net"},"d":{"h":"*.wmg-gardens.com"}},
+		{"o":{"h":"*.jorane.com"},"d":{"h":"*.html5shiv.googlecode.com"}},
+		{"o":{"h":"*.madsenmusik.de"},"d":{"h":"*.myplaydirectcrm.com"}},
```

Comment:
not required for me, the only problematic page is  /news/ and the rule below (sme-cdn.com) fixes it.